[FAB-130] Auto-scrolling PageToolbar

daystram opened the pull request.

nicolauscg reviewed (commented):
padding offset seems to be a bit off on a story with many pages, scrollbar is at top most position but does not show first page, look at http://localhost:3000/gallery/a7ea39b5-75ab-423a-ad8d-09ac7eef28cf/2a557fa5-b3f3-42f8-8eba-c5b660ad5b3f

all good otherwise

daystram commented:
> padding offset seems to be a bit off on a story with many pages, scrollbar is at top most position but does not show first page, look at http://localhost:3000/gallery/a7ea39b5-75ab-423a-ad8d-09ac7eef28cf/2a557fa5-b3f3-42f8-8eba-c5b660ad5b3f
> 
> all good otherwise

I have no clue why `align-content: space-evenly` breaks the alignement, so I reverted it. fixed
